sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -394,7 +399,7 @@ export class ApplicationStudentsController extends BaseController {
       programYearId: applicationProgramYear.programYear.id,
       formName: applicationProgramYear.programYear.formName,
       active: applicationProgramYear.programYear.active,
-    } as ApplicationWithProgramYearDto;
+    } as ApplicationWithProgramYearAPIOutDTO;
```

Comment:
same here https://github.com/bcgov/SIMS/pull/1636/files#r1062891943

Reply:
do we need `as ApplicationWithProgramYearAPIOutDTO;` here?

Reply:
I forgot to remove it.